### PR TITLE
Update test for data v4.9

### PR DESCRIPTION
**Pull request by calvinlu3**
This is related to https://github.com/oncokb/oncokb/issues/3570

### CODE REVIEW

File: core/src/test/resources/test_mutation_effect_summaries.tsv [hunk 1]
```diff
@@ -30,7 +30,7 @@ BRCA1	T100*	Likely Loss-of-function	The mutation effect description for truncati
...
+GATA3	T100*	Likely Loss-of-function	The mutation effect description for N-terminal truncating mutations in GATA3 is: The GATA3 1_307trunc mutations occur in the N-terminal region of GATA3 and result in the loss of the zinc finger 2 domain and the C-terminus. GATA3 truncating mutations in this range have been identified in breast cancer (PMID: 27251275). Truncated N-terminal transcripts of GATA3 have been identified as inactivating as measured by reduced transcriptional regulation of genes compared to wildtype and truncated C-terminal transcripts of GATA3 (PMID: 27588951). Truncated N-terminal transcripts of GATA3 are suggested to negatively impact the dimerization of the protein (PMID: 27588951, 31107186, 25410484).
```
Comment (zhx828) [inline core/src/test/resources/test_mutation_effect_summaries.tsv:33]: We should not use `1_307trunc mutations` here, but name of the string. I can fix later.

File: core/src/test/resources/test_variant_summaries.tsv [hunk 2]
```diff
@@ -30,7 +30,7 @@ BRCA1	T100*	Likely Oncogenic	The BRCA1 T100* is a truncating mutation in a tumor
 ### Truncating mutations in TSG that are not curated
 PPP2R2A	T100*	Unknown	The PPP2R2A T100* mutation has not specifically been reviewed by the OncoKB team, and therefore its biological significance is unknown.
 ### Truncating mutation in both
-GATA3	T100*	Likely Oncogenic	The GATA3 T100* is a truncating mutation; truncating mutations in this gene are considered likely oncogenic.
```
Comment (zhx828) [inline core/src/test/resources/test_variant_summaries.tsv:33]: Let's use gene `PTPN1`. The Truncating Mutations was removed from GATA3 in this release. The purpose of the test is to use oncogenicity from Truncating Mutations.

File: core/src/test/resources/test_variant_summaries.tsv [hunk 3]
```diff
@@ -42,6 +42,6 @@ BRCA1	Test-BRCA1 fusion	Likely Oncogenic	The Test-BRCA1 fusion leads to truncati
 ### Fusion in TSG that Truncating Mutations is not curated
 PPP2R2A	Test-PPP2R2A fusion	Unknown	The Test-PPP2R2A fusion has not specifically been reviewed by the OncoKB team, and therefore its biological significance is unknown.
 ### Fusion in both
-GATA3	Test-GATA3 fusion	Likely Oncogenic	The Test-GATA3 fusion leads to truncation of the GATA3 tumor suppressor gene and is considered likely oncogenic.
```
Comment (zhx828) [inline core/src/test/resources/test_variant_summaries.tsv:45]: Let's use PTPN1 and Test-PTPN1 here.

**Review by zhx828 — CHANGES_REQUESTED** [review]

**Comment by calvinlu3** [comment]
Thanks, you're comments make sense. 👍

**Review by zhx828 — APPROVED** [review]

**Comment by zhx828** [comment]
@calvinlu3 I'm gonna merge this PR so I can fix the summary on my end. Please send another PR if there is other change you like to include.

**Pull request merged**